### ISSUE TITLE
Cannot use object of type stdClass as array

### ISSUE THREAD

**Issue by kokgreat** [alexandreelise/j4x-api-examples#5]
Good day! 
Thanks for this sample API. 
I'm trying to use it and I get a message: "Cannot use object of type stdClass as array" 

I changed the error output code to this: echo $e->GetMessage() . ' - ' . $e->getline() . PHP_EOL; and I see that the error in line 229: $storage[$DataKey] = ['mightExists' => $decodedJsonOutput->errors[0]->code === 400, 'decodedDataString' => $decodedDataString];

At the same time, the rest of the code works correctly and the data in Joomla is updated, although I haven't checked all the code options yet :-)

https://github.com/alexandreelise/j4x-api-examples/blob/d70dbeeb16dffaaac3e8a83b38d79541df86ec77/using-raw-php/smart-add-edit-to-multiple-sites-from-csv-url.php#L229

Joomla 4.2.8, PHP Version 8.1.11

**Comment by alexandreelise** [comment]
Hi @kokgreat,

Thanks for reporting what seems to be an issue. Unfortunately I cannot investigate the potential problem yet. Maybe some other contributor can help you for now. Sorry for the inconvenience.

Anyway, enjoy your day.

**Comment by alexandreelise** [comment]
Hi @kokgreat,

I looked into what you flagged as an issue and could not reproduce it.

I suggests that you use the latest stable Joomla 4 version which is 4.2.9 at the time I am writing this reply. And latest php 8.1 version which is 8.1.17 et the time of writing this reply. Although, that might not have been the source of the issue you found which might be totally relevant in your context, maybe you don't have the exact datasource I provided in the example. Try that before making your own datasource. I am closing this issue for now. Have a great weekend.
